src/components/Comments/Comment.js

```diff
@@ -253,12 +251,14 @@ const Comment = ({
           </TouchableOpacity>
         )}
 
-        <ButtonHightlight
-          onLongPress={handleOnLongPress}
-          style={[styles.btnBlock(comment.user.id === yourselfId), styles.btn]}
-          onPress={() => onBlockComponent(comment)}>
-          <IconEn name="block" size={15.02} color={COLORS.balance_gray} />
-        </ButtonHightlight>
+        {!comment?.is_you && (
+          <ButtonHightlight
+            onLongPress={handleOnLongPress}
+            style={[styles.btnBlock(comment.user.id === yourselfId), styles.btn]}
+            onPress={() => onBlockComponent(comment)}>
+            <IconEn name="block" size={15.02} color={COLORS.balance_gray} />
+          </ButtonHightlight>
+        )}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Good job on adding a condition to hide the block button if the comment is from the current user. This improves the user experience by preventing users from blocking themselves.



<!-- This is an auto-generated comment by OSS CodeRabbit -->